src/styles/main.less

```diff
@@ -748,11 +748,28 @@ i[class*="gender-icon-"] {
   }
 }
 
+button.btn-circle {
```

Comment:
Is that on a specific theme? It doesn't look low to me
![Screenshot_2023-05-02_01-15-24](https://user-images.githubusercontent.com/4183969/235553393-b405b981-9b5f-424f-b31f-84d1145da15c.png)

Reply:
It looks like that in my game, and i feel like it's off? We might need a third person...

Reply:
pixel counting says the center of the cross is 22 pixels from the top, left, and right, and 21 from the bottom. it is as centered as it can be given it's dimensions and the dimensions of the circle.

Reply:
maybe I can add another pixel of height?

Reply:
I wanted to suggest that, but i thought it would be a lot of work for something no one else notices... But you can do it if you want to